scripts/audit_large_rust_files.py

```
@@ -0,0 +1,84 @@
+
+
+def rust_files(root: str):
+    for dirpath, dirnames, filenames in os.walk(root):
+        dirnames[:] = [d for d in dirnames if d not in EXCLUDE_DIRS]
+        for name in filenames:
+            if name.endswith(".rs"):
+                yield os.path.join(dirpath, name)
+
+
+def line_count(path: str) -> int:
+    with open(path, "r", encoding="utf-8") as handle:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)